src/parse.rs

```diff
@@ -764,10 +764,11 @@ fn parse_csi_normal_mouse(buffer: &[u8]) -> Result<Option<Event>> {
     let (kind, modifiers) = parse_cb(cb)?;
 
     // See http://www.xfree86.org/current/ctlseqs.html#Mouse%20Tracking
-    // The upper left character position on the terminal is denoted as 1,1.
-    // Subtract 1 to keep it synced with cursor
-    let cx = u16::from(buffer[4].saturating_sub(32)) - 1;
-    let cy = u16::from(buffer[5].saturating_sub(32)) - 1;
+    // Mouse positions are encoded as (value + 32), but the upper left
+    // character position on the terminal is denoted as 1,1.
+    // So, we need to subtract 32 + 1 (33) to keep it synced with the cursor.
+    let cx = u16::from(buffer[4].saturating_sub(33));
+    let cy = u16::from(buffer[5].saturating_sub(33));
```

Comment:
I'll do some testing later but I'm pretty sure this patch is correct. From the linked doc:

> Parameters (such as pointer position and button number) for all mouse tracking escape sequences generated by xterm encode numeric parameters in a single character as value+32. For example, ! specifies the value 1. The upper left character position on the terminal is denoted as 1,1.